CHANGELOG.md

```diff
@@ -3,6 +3,87 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0
+
+_Nov 12, 2021_
+
+🎉 This is the new stable release of the data grid component 🎉!
+
+If you want to migrate the DataGrid or DataGridPro from v4 to v5, take a look at the [migration guide](https://mui.com/components/data-grid/migration-v4/).
+This version is fully compatible with `@mui/material@5.X` and can be used with `@material-ui/core@4.x` with some additional steps as described [here](https://mui.com/components/data-grid/migration-v4/#using-mui-x-v5-with-mui-core-v4)
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- ⌨️ Enhance keyboard navigation when pagination is enabled
+- 👁 Better support for flex columns
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.0.0` / `@mui/x-data-grid-pro@v5.0.0`
+
+#### Breaking changes
+
+- [DataGrid] The following CSS classes were renamed to follow the internal convention:
+
+  - `MuiDataGrid-columnsContainer` was renamed to `MuiDataGrid-columnHeaders`
+  - `MuiDataGrid-columnHeaderWrapper` was renamed to `MuiDataGrid-columnHeadersInner`
+  - The `scroll` class applied to `MuiDataGrid-columnHeaderWrapper` was renamed to `MuiDataGrid-columnHeadersInner--scrollable`
+
+- [DataGrid] The `props.components.Checkbox` and `props.componentsProps.checkbox` props were renamed to `props.components.BaseCheckbox` and `props.componentsProps.baseCheckbox` respectively.
+
+  As a first step for [#3066](https://github.com/mui-org/material-ui-x/issues/3066), these slots were renamed to clearly indicate that they are meant to replace a core component.
+
+  ```diff
+  <DataGrid
```

Comment:
This is not the standard way git diff outputs, there is always a reserved first char for line (one of ` `, or `-`, or `+`)
```suggestion
   <DataGrid
```